client/src/components/game/deck/Deck.jsx

```diff
@@ -0,0 +1,33 @@
+import './deck.scss';
+
+import React from 'react';
+
+import { CardReverse } from '../cards/CardReverse';
+
+const Deck = ({ color, cardsLeft, cardsMax }) => {
+  let cardsToRender = cardsLeft > 10 ? 10 : cardsLeft;
+
+  let cardElements = new Array(cardsToRender);
+  for (let i = 0; i < cardsToRender; i++) {
+    let element = (
```

Comment:
If variable isn't changing then declaration with `const` is more readable. Declaring variables with `let` implies its later change